### PR TITLE
Task/implement ng2 select in user form

### PR DESCRIPTION
**Pull request by JuanGro**
#### What's this PR do?
Change the multipicker to ng2-select

#### Where should the reviewer start?
With the new groups filed form.

#### How should this be manually tested?
Creating and updating a user.

#### Any background context you want to provide?
No.

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: src/app/admin/users/user-form.component.ts [hunk 1]
```diff
@@ -50,6 +51,14 @@ export class UserFormComponent implements OnChanges {
   public active: boolean = true;
   // Variable to show the role of the contact selected.
   public array: string = '';
+  // Variable to save which group is selecting the user in the Ng2-Select.
+  public value: Array<any>;
```
Comment (EduardoVaca) [inline src/app/admin/users/user-form.component.ts]: Pick a better variable name, like "currentGroupNameList"

File: src/app/admin/users/group-model.ts [hunk 2]
```diff
@@ -15,3 +15,12 @@ export class Group {
     this.name = json && json.name;
   }
 }
+
+/**
+* Class that declares all the attributes used in the roles for ng-select field and use them
+* to submit the user object to the API.
+**/
+export class Role {
```
Comment (EduardoVaca) [inline src/app/admin/users/group-model.ts]: Change this class name to GroupString

File: src/app/admin/users/user-form.component.ts (hunk 1 above)
Comment (JuanGro) [inline src/app/admin/users/user-form.component.ts]: We, pero no vale

Comment (JuanGro) [inline src/app/admin/users/user-form.component.ts]: Done

File: src/app/admin/users/group-model.ts (hunk 2 above)
Comment (JuanGro) [inline src/app/admin/users/group-model.ts]: Done

**Pull request merged**